Mathlib/GroupTheory/MonoidLocalization/Basic.lean

```diff
@@ -1369,4 +1369,17 @@ lemma mk_div_mk (m₁ m₂ : M) (s₁ s₂ : (⊤ : Submonoid M)) :
     mk m₁ s₁ / mk m₂ s₂ = .mk (m₁ * s₂) ⟨s₁ * m₂, Submonoid.mem_top _⟩ := by
   simp [div_eq_mul_inv, mk_mul]; rfl
 
+/-- A monoid homomorphism from a monoid `M` to a group `G` lifts to a group homomorphism from the
+Grothendieck group of `M` to `G`. -/
+@[to_additive
+"A monoid homomorphism from a monoid `M` to a group `G` lifts to a group homomorphism from the
+Grothendieck group of `M` to `G`."]
+noncomputable def lift (f : M →* G) : GrothendieckGroup M →* G :=
```

Comment:
I'd argue for
```suggestion
noncomputable def lift : (M →* G) ≃ (GrothendieckGroup M →* G) :=
```
though we're not completely consistent on whether we call this `lift` or `liftEquiv`.